openmdao/recorders/test/test_hdf5.py

```diff
@@ -7,14 +7,38 @@
 import unittest
 import time
 
+import numpy as np
+
 from six.moves import zip
 from six import iteritems
 
+from openmdao.api import ScipyOptimizer
 from openmdao.core.problem import Problem
 from openmdao.test.converge_diverge import ConvergeDiverge
 from openmdao.test.example_groups import ExampleGroup
-from openmdao.util.record_util import format_iteration_coordinate
+from openmdao.test.sellar import SellarDerivativesGrouped
 from openmdao.test.util import assert_rel_error
+from openmdao.util.record_util import format_iteration_coordinate
+
+# check that pyoptsparse is installed
+# if it is, try to use SLSQP
+OPT = None
+OPTIMIZER = None
+
+try:
+    from pyoptsparse import OPT
+    try:
+        OPT('SLSQP')
+        OPTIMIZER = 'SLSQP'
+    except:
+        pass
+except:
+    pass
+
+if OPTIMIZER:
+    from openmdao.drivers.pyoptsparse_driver import pyOptSparseDriver
+
```

Comment:
oh! I didn't know about that. I just stole that code from somewhere else in the framework. I forgot where. Will give your solution a try. Thanks